oauth2/.snapshots/TestAuthCodeWithMockStrategy-strategy=jwt-case=5-description=should_pass_with_prompt=login_when_authentication_time_is_recent-should_call_refresh_token_hook_if_configured-hook=new.json

```diff
@@ -17,10 +17,7 @@
           "hooked": "legacy"
         }
       },
-      "headers": {
-        "extra": {
-        }
-      },
+      "headers": null,
```

Comment:
Please keep the headers.extra keys, as changing them to null will potentially break webhook receivers.